lib/utils.js

```diff
@@ -45,6 +45,12 @@ function normalizeBinary (binaryPath, platform, arch) {
 
       if (channelNames.indexOf(binaryPath) !== -1) {
         result = findMacAppByChannel(binaryPath);
+
+        // recent versions (as of July 2020) of Firefox Developer Edition use
+        // 'org.mozilla.firefoxdeveloperedition' as the CFBundleName
+        if (!result && binaryPath === "deved") {
```

Comment:
`deved` had never been an alias, it was recently introduced in #44.

Let's fix this specific part by exchanging "deved" and "firefoxdeveloperedition" in the implementation.